src/index.js

```diff
@@ -21,24 +21,24 @@ document.querySelector('form').addEventListener('submit', (e) => {
   UiTodo.deleteData();
 });
 document.getElementById('to-do-container').addEventListener('click', (e) => {
-  Storage.editInput(
-    e.target.parentElement.parentElement.children[4],
-    e.target.parentElement,
-    e.target.parentElement.parentElement,
-    e.target.parentElement.parentElement.children[0].children[0],
-  );
+  const eventTarget = e.target;
   UiTodo.trashTodo(e.target);
+
   if (e.target.classList.contains('check')) {
     Storage.checkboxCompleted(
-      e.target.parentElement.parentElement.children[4],
-      e.target.checked,
+      eventTarget.parentElement.children[4],
+      eventTarget.checked,
     );
-    e.target.parentElement.parentElement.children[2].children[0].classList.toggle(
-      'strike-through',
+    eventTarget.parentElement.children[1].classList.toggle('strike-through');
+    Storage.editInput(
+      e.target.parentElement.parentElement.children[4].textContent,
```

Comment:
[REQUIRED] There are some problems with your `delete`, `clear all completed` and `removing from local storage` functionalities. Whenever a todo is deleted it should be removed from the local storage but it comes back when I refresh. Your `clear al competed` button does not clear checked todos

![todo-1](https://user-images.githubusercontent.com/62938735/202677836-3fe86a7c-8efc-49a3-9a02-15e581e13bb3.gif)